packages/create-react-native-library/src/index.ts

```diff
@@ -557,6 +557,7 @@ async function create(argv: yargs.Arguments<any>) {
             `${project.charAt(0).toUpperCase()}${project
               .replace(/[^a-z0-9](\w)/g, (_, $1) => $1.toUpperCase())
               .slice(1)}`,
+      year: new Date().getFullYear(),
```

Comment:
It would make more sense if the `year` could be top-level instead of inside the `project`. Could you place the `year` directly inside the options?

Then you can use it as `<%- year -%>` in the template.